### PR TITLE
Refs #10970: apply backwards compatible rails 4 changes.

### PR DESCRIPTION
**Pull request by waldenraines**
Preemptive changes for future rails 4 upgrade that are
backwards compatible.

http://projects.theforeman.org/issues/10970

### CODE REVIEW

File: app/assets/stylesheets/katello/contents.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import "katello/katello_base";
-@import "jquery.timepickr";
+@import "katello/jquery.timepickr";
```
Comment (ehelms) [inline app/assets/stylesheets/katello/contents.scss]: You might could even remove this -- I am not sure where timepickr would be used in the old UI

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by waldenraines** [comment]
[test]

**Comment by waldenraines** [comment]
[test]

**Comment by jlsherrill** [comment]
APJ from me

**Comment by waldenraines** [comment]
Latest commit was just a rebase to include #5331.

**Pull request merged**